woocommerce-services.php

```diff
@@ -914,22 +874,32 @@ public function attach_hooks() {
 			$this->taxjar->init();
 			$this->paypal_ec->init();
 
-			// Primary condition to initiate shipping.
-			if ( ! $this->is_wc_shipping_activated() ) {
+			/*
+			 * Only register shipping label-related logic if WC Shipping is not active.
+			 */
```

Comment:
Not something we should spend time on, but technically this is a single inline comment 😄 

```suggestion
			// Only register shipping label-related logic if WC Shipping is not active.
```